Apply an edit pass to fix several docs issues (#182)
globus/globus-automate-client: docs/source/authoring_flows.rst

Authoring Flows for the Globus Flows Service
============================================

*Globus Flows* provides users with the ability to easily define
compositions of actions (henceforth referred to as *flows*) to perform a single,

> logical operation. Definition of such flows requires an easy to read, author,
> and potentially visualize method of defining the flows. For this purpose, the

Reviewer: ```suggestion
logical operation. The method for defining such flows must be easy to read, author,
and potentially visualize. For this purpose, the
```
